### PR TITLE
use pipe clients to send requests to tablets

### PR DESCRIPTION
**Pull request by andrewstalin**
### Changelog entry

Вместо PipeCache используем PipeClient для отправки запросов таблеткам ColumnShard.

### Changelog category

* Not for changelog (changelog entry is not required)

### CODE REVIEW

File: ydb/core/statistics/service/ut/ut_service.cpp [hunk 1]
```diff
@@ -154,235 +154,382 @@ std::unordered_map<ui32, ui32> InitializeRuntime(TTestActorRuntime& runtime, ui3
...
+            auto recipient = indexToActorMap[nodeIdToIndexMap[ev->Sender.NodeId()]];
+            runtime.Send(new IEventHandle(recipient, ev->Sender,
+                new TEvTabletPipe::TEvClientConnected(tabletId, NKikimrProto::OK, ev->Sender, ev->Sender,
+                true, false, 0), 0, ev->Cookie), nodeIdToIndexMap[ev->Sender.NodeId()], true);
+            ev.Reset();
         }));
-        observers.emplace_back(runtime.AddObserver<TEvPipeCache::TEvGetTabletNodeResult>([&](TEvPipeCache::TEvGetTabletNodeResult::TPtr& ev) {
-            if (nodeIdToIndexMap.find(ev->Recipient.NodeId()) != nodeIdToIndexMap.end()) {
-                --inFlight;
+        observers.emplace_back(runtime.AddObserver([&](TAutoPtr<IEventHandle>& ev) {
```
Comment (azevaykin) [inline ydb/core/statistics/service/ut/ut_service.cpp:371]:
```suggestion
        observers.emplace_back(runtime.AddObserver([&](TEvStatistics::TEvStatisticsRequest& ev) {
```

File: ydb/core/statistics/service/service_impl.cpp [hunk 2]
```diff
@@ -969,31 +894,104 @@ class TStatService : public TActorBootstrapped<TStatService> {
...
+        for (const auto& tag : AggregationStatistics.ColumnTags) {
+            columnTags->Add(tag);
+        }
+
+        NTabletPipe::SendData(SelfId(), clientId, request.release(), AggregationStatistics.Round);
+    }
+
+    void OnTabletError(ui64 tabletId) {
+        LOG_DEBUG_S(TlsActivationContext->AsActorContext(), NKikimrServices::STATISTICS,
+            "Tablet " << tabletId << " is not local.");
```
Comment (azevaykin) [inline ydb/core/statistics/service/service_impl.cpp:916]: Is this message right when it is called from 958 line?

Comment (andrewstalin) [reply ydb/core/statistics/service/service_impl.cpp:916]: > Is this message right when it is called from 958 line?

What's bothering you? It is assumed that the tablet is local and if there are problems connecting to it, it has probably gone away in the process. The aggregator will request information about it and try again.

Comment (azevaykin) [reply ydb/core/statistics/service/service_impl.cpp:916]: May be there could be more error reasons, not only "is not local".

Comment (andrewstalin) [reply ydb/core/statistics/service/service_impl.cpp:916]: What's the point? What will the aggregator do with them?

Comment (azevaykin) [reply ydb/core/statistics/service/service_impl.cpp:916]: The point is only the message text)

File: ydb/core/statistics/service/ut/ut_service.cpp (hunk 1 above)
Comment (andrewstalin) [reply ydb/core/statistics/service/ut/ut_service.cpp:371]: What are you suggesting to do here?

**Review by azevaykin — APPROVED** [review]

**Pull request merged**